lms/djangoapps/experiments/tests/test_views.py

```diff
@@ -263,8 +263,8 @@ def _cross_domain_post(self, csrf_token, data, referer=CROSS_DOMAIN_REFERER):
 
     def test_white_list_contents_with_cors_header_version(self, *args):  # pylint: disable=unused-argument
         """ Verify that with django-cor-header<3 it loads list without scheme. """
```

Comment:
fix doc string.

Reply:
What is the point of this test?  I don't understand why it was added and why we need to keep it?

Reply:
[Here](https://github.com/edx/edx-platform/pull/28588) added conditions to pick list as per version.
So just added a test to make sure things are working as expected.

Reply:
After successful deployment  I will remove following things
1. Move content of  `CORS_ORIGIN_WHITELIST_WITH_SCHEME`  into `CORS_ORIGIN_WHITELIST` and remove it from all edge, prod and sandbox configs.
2. Remove the conditions from `lms` and `cms` production.py
3. Remove this test also.